front/src/pages/settings/data-model/SettingsObjectFieldEdit.tsx

```diff
@@ -16,15 +49,33 @@ export const SettingsObjectFieldEdit = () => {
             links={[
               { children: 'Objects', href: '/settings/objects' },
               {
-                children: `${objectSlug}`,
+                children: activeObject.labelPlural,
                 href: `/settings/objects/${objectSlug}`,
               },
-              {
-                children: `${fieldName}`,
-              },
+              { children: activeField.label },
             ]}
           />
         </SettingsHeaderContainer>
+        <SettingsObjectFieldFormSection
+          disabled={!activeField.isCustom}
+          name={activeField.label}
+          description={activeField.description ?? undefined}
+          iconKey={activeField.icon ?? undefined}
+          onChange={() => undefined}
```

Comment:
Is it normal that the input changes are not handled? Maybe you want to do that in another PR

Reply:
Yes this is addressed in this PR: https://github.com/twentyhq/twenty/pull/2245